app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<section class="login">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
```

Comment:
- [ ] I cannot sign up to your app. You are required to have a `name` field in your form. You don't have it so a user cannot sign up. Kindly fix that and test your app before requesting another review.

- [ ] Since I could not sign up I could not test other requirements after that. Kindly go through the project guidelines and ensure everything works as required.